src/test/java/seedu/progresschecker/commons/util/AppUtilTest.java

```diff
@@ -15,7 +15,7 @@
 
     @Test
     public void getImage_exitingImage() {
-        assertNotNull(AppUtil.getImage("/images/address_book_32.png"));
+        assertNotNull(AppUtil.getImage("/images/progress_checker_32.png"));
```

Comment:
Might want to change this back to ```address_book_32.png``` for the image file to be valid. P.S. Not recommended to rename the image file to solve this instead because the image does not belong to us so we shouldn't claim it.

Reply:
changing the image name as well as its occurrence in the copywrite file is how we give credit to the original creator.
and note that the image name "address_book_32.png" is already not the original file name (ie. changed by the software developer off addressbook-level4).